sentry/src/main/java/io/sentry/SentryUtilDate.java

```diff
@@ -0,0 +1,22 @@
+package io.sentry;
+
+import java.util.Date;
+import org.jetbrains.annotations.NotNull;
+
+public final class SentryUtilDate extends SentryDate {
```

Comment:
`l`: I think this class is only used in tests - should we make it internal? or should we move it to the `test` source set? Or do we even need it?